Description: Upstream changes introduced in version 0.0.1rc1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 baruwa (0.0.1rc1-1) unstable; urgency=low
 .
   * Initial release (Closes: #9999)
   * Switch to dpkg-source 3.0 (quilt) format
 .
 The person named in the Author field signed this changelog entry.
Author: Andrew Colin Kissa <andrew@topdog.za.net>
Bug-Debian: http://bugs.debian.org/9999

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- baruwa-0.0.1rc1.orig/src/baruwa.egg-info/PKG-INFO
+++ baruwa-0.0.1rc1/src/baruwa.egg-info/PKG-INFO
@@ -10,34 +10,34 @@ Description: 
         ======
         Baruwa
         ======
-        Baruwa (swahili for letter or mail) is a `mailwatch <http://mailwatch.sf.net/>`_ inspired
-        web 2.0 `MailScanner <http://www.mailscanner.info/>`_ front-end.
+        Baruwa (swahili for letter or mail) is a `mailwatch <http://mailwatch.sf.net/>`_ inspired 
+        web 2.0 `MailScanner <http://www.mailscanner.info/>`_ front-end. 
         
-        It provides an easy to use interface for users to view details of messages processed by
-        MailScanner as well as perform operations such as releasing quarantined messages,
-        spam learning, whitelisting and blacklisting addresses etc. Baruwa has implemented web 2.0
+        It provides an easy to use interface for users to view details of messages processed by 
+        MailScanner as well as perform operations such as releasing quarantined messages, 
+        spam learning, whitelisting and blacklisting addresses etc. Baruwa has implemented web 2.0 
         features (AJAX) where deemed fit, graphing is also implemented on the client side using SVG.
         
-        It also provides reporting functionality with an easy to use query builder, results can be
+        It also provides reporting functionality with an easy to use query builder, results can be 
         displayed as message lists or graphed as colorful and pretty graphs.
         
-        Baruwa is written in Python using the Django Framework and MySQL for storage, it is released
+        Baruwa is written in Python using the Django Framework and MySQL for storage, it is released 
         under the GPLv2
         
         ========
         Features
         ========
-        + AJAX refreshed recent messages listing
-        + Detailed message view with AJAX enabled message processing (quarantine release/delete,spam learning,white/black listing)
-        + AJAX powered Full/Quarantine messages listings
-        + Reporting view with AJAX enabled query builder
-        + Interactive SVG graphs
-        + Multi user profiles
-        + User profile aware white/blacklist management
-        + REST API
-        + Easy plug-in authentication to external authentication systems (Mailwatch, POP3, IMAP and SMTP supported out of the box)
-        + Tools for housekeeping tasks (quarantine management, rule updates, quarantine notifications, etc)
-        + Works both with and without Javascript enabled (some features will have degraded performance)
+            + AJAX refreshed recent messages listing
+            + Detailed message view with AJAX enabled message processing (quarantine release/delete,spam learning,white/black listing)
+            + AJAX powered Full/Quarantine messages listings
+            + Reporting view with AJAX enabled query builder
+            + Interactive SVG graphs
+            + Multi user profiles
+            + User profile aware white/blacklist management
+            + REST API
+            + Easy plug-in authentication to external authentication systems (Mailwatch, POP3, IMAP and SMTP supported out of the box)
+            + Tools for housekeeping tasks (quarantine management, rule updates, quarantine notifications, etc)
+            + Works both with and without Javascript enabled (some features will have degraded performance)
         
         ===========
         Screenshots
@@ -47,20 +47,20 @@ Description: 
         ============
         Requirements
         ============
-        + Python >= 2.4
-        + Django >= 1.1.1
-        + MySQLdb >= 1.2.1p2
-        + GeoIP
-        + iPy
-        + Any Web server that can run Django (Apache/mod_wsgi recommended)
-        + MySQL
+            + Python >= 2.4
+            + Django >= 1.1.1
+            + MySQLdb >= 1.2.1p2
+            + GeoIP
+            + iPy
+            + Any Web server that can run Django (Apache/mod_wsgi recommended)
+            + MySQL
         
         ============
         Installation
         ============
         Baruwa is installed in the usual way::
         
-        python setup install
+            python setup install
         
         =============
         Documentation
